docker-compose.override.yml.example

```diff
@@ -1,12 +1,40 @@
 version: '3'
 
 services:
+  rabbitmq:
```

Comment:
Shouldn't it just be called `rabbit` like in `docker-compose.yml` below?

Reply:
yes.